### DISCUSSION TITLE
Deployment

### DISCUSSION THREAD

**Discussion by hendellcosta** [vercel/community#2857] [Help] [answered]
### Summary

Hey! In some projects I do, I can't deploy to vercel, and when I can deploy, after some changes i stop working, and it seems to be a "Failed production deployment" and I don't know why it happens.
![Screenshot 2023-06-14 034728](https://github.com/vercel/community/assets/86265741/d70f97fa-6b45-45b9-85ad-8a7c75fdf632)

### Steps to Reproduce

When I change my front-end it starts getting error, and sometimes in the beggining of my app.

**Comment by swarnava** [comment]
Can you share the deployment url?

**Reply by hendellcosta** [reply]
I can't get the link because I can't deploy.

**Reply by swarnava** [reply]
Is this a notification email? There should be black button  with `View deployment` to find deployment link

**Reply by hendellcosta** [reply]
![image](https://github.com/vercel/community/assets/86265741/f7c05af0-cf0b-4442-9c54-9bfedbfd88c0)

do you know why?

**Reply by swarnava** [reply, marked as answer]
This is 90% related to code you have deployed. Are you able to run build command locally?

**Reply by hendellcosta** [reply]
You were right, thank you!
